src/modules/addRemove.js

```diff
@@ -0,0 +1,174 @@
+import Task from './task.js';
+
+class displayTask {
+  static displayTasks() {
+    const tasks = Store.getTasks();
+    tasks.forEach((task) => displayTask.addTaskToList(task));
+  }
+
+  static addTaskToList(task) {
+    const elementList = document.querySelector('.list');
+    const listItem = document.createElement('li');
+    listItem.setAttribute('class', 'item');
+
+    listItem.innerHTML = `<input type="checkbox" id='${task.id}' name="taskStatus" class = 'taskStatus'>
+                    <p class='description edit' contenteditable='true' id='${task.id}'>${task.description}</p>
+                    <i class="fas fa-trash-alt remove-btn" id='${task.id}'></i>`;
+
+    elementList.appendChild(listItem);
+  }
+
+  static deleteTask(el) {
+    if (el.classList.contains('remove-btn')) {
+      el.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
+    document.querySelector('#inputTask').value = '';
+  }
+}
+
+class Store {
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+
+    return tasks;
+  }
+
+  static addTask(task) {
+    const tasks = Store.getTasks();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static removeTask(elem) {
+    if (elem.classList.contains('remove-btn')) {
+      const tasks = Store.getTasks();
+      if (tasks.length === 1) {
+        tasks.pop();
+      } else {
+        tasks.splice(parseInt(elem.id), 1);
+
+        tasks.forEach((element, index) => {
+          element.id = index + 1;
+        });
+      }
+      index = tasks.length;
+
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
+
+  static modifyTask(elem) {
+    const tasks = Store.getTasks();
+    console.log(elem.id);
+    tasks[elem.id].description = elem.innerHTML;
+
+    tasks.forEach((element, index) => {
+      element.id = index + 1;
+    });
+    index = tasks.length;
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
```

Comment:
- [ ] Your edit functionality does not work as expected. When I edit a task it updates a different task in the `localStorage`. From the screenshots below, you can see that I am editing `first task` but a different task is being updated, also an entirely different task shows up in the storage. The appropriate task should be edited. Kindly fix this.

  The first task is edited | Page is reloaded - an unknown task is added
  :---:|:---:
  ![shing-1](https://user-images.githubusercontent.com/36542986/165675158-e422c687-49ff-40f8-acc7-a25893be4bf1.jpg) | ![shing-2](https://user-images.githubusercontent.com/36542986/165675163-b9a5a9d7-08c8-467b-9d75-2dea5ce9355b.jpg)